### DISCUSSION TITLE
Regarding line feeds at the end of files

### DISCUSSION THREAD

**Discussion by doomori** [mozman/ezdxf#639] [Issues]
Thanks for the great library.

I have a DXF from a customer that contains an extra newline at the end of the file.
Sometimes they send me DXFs with extra line breaks at the end of the file.
DXFStructureError occurs and I have to use
DXFStructureError and I am having trouble. 

![image](https://user-images.githubusercontent.com/57756511/155822456-8d8830f2-0b46-43a3-8126-eeab6e2c1deb.png)

3345
3346
There is an extra line break on line 3345 and 3346.

 I try to delete the end of the file manually, but
Is there anything I can do?

thank you.

**Comment by mozman** [comment]
see issue #640
